src/app/projects/[projectId]/Proposals.tsx

```diff
@@ -91,31 +90,74 @@ export default function Proposals({ projectId }: { projectId: string }) {
     }
   };
 
-  const handleExportToDocx = () => {
+  const handleExportToDocx = async () => {
+    const processContent = (content: string) => {
+      const paragraphs: Paragraph[] = [];
+      const lines = content.split("\n");
+  
+      lines.forEach((line) => {
+        if (line.startsWith("### ")) {
```

Comment:
considerar usar line.includes
```suggestion
        if (line.includes("### ")) {
```